<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color: #ffffff;
}
QLineEdit{
background-color: #5EBDB2;
border-radius: 5px;
color: #ffffff;
}
QListWidget{
background-color: #c8e5e3;
border-radius: 10px;
}
QLabel{
color: #142b3b;
}
QGroupBox{
color: #1792a4;
border-radius: 10px;
background-color: #e8e8e7;
}
QPushButton{
 	background-color: #142b3b;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 13px;
    min-width: 7em;
    padding: 3px;
	color:#c8e5e3;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>&quot;Call Me&quot;, (Maybe)?</family>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Buscar cliente</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>611</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Grande</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Introducir datos para buscar</string>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>&quot;Call Me&quot;, (Maybe)?</family>
      </font>
     </property>
     <property name="text">
      <string>Nombre</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>&quot;Call Me&quot;, (Maybe)?</family>
      </font>
     </property>
     <property name="text">
      <string>Apellidos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Buscar">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>122</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>611</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Grande</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Datos</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>&quot;Call Me&quot;, (Maybe)?</family>
      </font>
     </property>
     <property name="text">
      <string>Nombre</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>&quot;Call Me&quot;, (Maybe)?</family>
      </font>
     </property>
     <property name="text">
      <string>Apellidos</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>80</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>&quot;Call Me&quot;, (Maybe)?</family>
      </font>
     </property>
     <property name="text">
      <string>Dirección</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>&quot;Call Me&quot;, (Maybe)?</family>
      </font>
     </property>
     <property name="text">
      <string>Localidad</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>40</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>20</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>&quot;Call Me&quot;, (Maybe)?</family>
      </font>
     </property>
     <property name="text">
      <string>Fecha
nacimiento</string>
     </property>
    </widget>
    <widget class="QListWidget" name="lista">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>591</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>60</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>&quot;Call Me&quot;, (Maybe)?</family>
      </font>
     </property>
     <property name="text">
      <string>Días límite</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_8">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>80</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>&quot;Call Me&quot;, (Maybe)?</family>
      </font>
     </property>
     <property name="text">
      <string>Fecha
alta</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>&quot;Call Me&quot;, (Maybe)?</family>
      </font>
     </property>
     <property name="text">
      <string>Fecha
baja</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_9">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_10">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>120</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>190</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>&quot;Call Me&quot;, (Maybe)?</family>
       <pointsize>19</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Últimos pagos</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_11">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>120</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>110</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>&quot;Call Me&quot;, (Maybe)?</family>
      </font>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>&quot;Call Me&quot;, (Maybe)?</family>
      </font>
     </property>
     <property name="text">
      <string>NIF</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_12">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>&quot;Call Me&quot;, (Maybe)?</family>
    </font>
   </property>
   <widget class="QMenu" name="menuRegistro">
    <property name="title">
     <string>Registro</string>
    </property>
    <addaction name="actionRegistrar"/>
   </widget>
   <widget class="QMenu" name="menuClientes">
    <property name="title">
     <string>Clientes</string>
    </property>
    <addaction name="facturas"/>
    <addaction name="actionCobrar_mes"/>
   </widget>
   <widget class="QMenu" name="menuInventario">
    <property name="title">
     <string>Inventario</string>
    </property>
    <addaction name="inventario"/>
   </widget>
   <addaction name="menuRegistro"/>
   <addaction name="menuClientes"/>
   <addaction name="menuInventario"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="font">
    <font>
     <family>&quot;Call Me&quot;, (Maybe)?</family>
    </font>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCrear_tarjeta">
   <property name="text">
    <string>Crear tarjeta</string>
   </property>
   <property name="font">
    <font>
     <family>&quot;Call Me&quot;, (Maybe)?</family>
    </font>
   </property>
  </action>
  <action name="facturas">
   <property name="text">
    <string>Cobros y facturas</string>
   </property>
  </action>
  <action name="actionCobrar_mes">
   <property name="text">
    <string>Consultar</string>
   </property>
  </action>
  <action name="actionRegistrar">
   <property name="text">
    <string>Registrar</string>
   </property>
  </action>
  <action name="inventario">
   <property name="text">
    <string>Consultar inventario</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
